src/server.js

```diff
@@ -280,7 +282,73 @@ app.get(
   }
 );
 
+app.get(
```

Comment:
I'm curious as to why the API is structured like this:
```
/api/${ADDRESS}/geocode
```
If I understand correctly, this would mean, to call the API, the client would do:
```
/api/20 Foobar St/geocode
```
(with 20 Foobar St) being encoded

This is rather unorthodox. I think it is best to have the API structured as follows:
```
/api/geocode?address=20 foobar st
```
In this case, 20 foobar St. is a query string URL parameter.

Reply:
Done.